karma.conf.ts

```diff
@@ -103,19 +103,19 @@ function makeBuildNumber() {
 }
 
 function setupLocal(config: Config) {
-  const files = ['src/**/*.ts', 'tests/**/*.ts', 'dist/fp.min.js']
+  const files = [
+    // The polyfills are required for old supported browsers.
+    // They should be removed when the old browser support is dropped.
+    'node_modules/promise-polyfill/dist/polyfill.js',
```

Comment:
The polyfill is not required because IE11 isn't supported anymore (if guys agree).

@Romchela, please confirm dropping support of some old browsers. This PR changes the list to [the browsers supported by FingerprintJS](https://github.com/fingerprintjs/fingerprintjs/blob/master/docs/browser_support.md).

Reply:
Let's use the same browser list which main JS Agent follows